src/main/scala/nl.knaw.dans.easy.multideposit/model/Deposit.scala

```diff
@@ -15,26 +15,37 @@
  */
 package nl.knaw.dans.easy.multideposit.model
 
+import java.util.UUID
+
 import better.files.File
 
 case class Instructions(depositId: DepositId,
                         row: Int,
                         depositorUserId: DepositorUserId,
                         profile: Profile,
-                        baseUUID: Option[BaseUUID]= Option.empty,
+                        baseUUID: Option[BaseUUID] = Option.empty,
                         metadata: Metadata = Metadata(),
                         files: Map[File, FileDescriptor] = Map.empty,
                         audioVideo: AudioVideo = AudioVideo()) {
   def toDeposit(fms: Seq[FileMetadata] = Seq.empty): Deposit = {
-    Deposit(depositId, row, depositorUserId, profile, baseUUID, metadata, fms, audioVideo.springfield)
+    Deposit(depositId = depositId,
+      row = row,
+      depositorUserId = depositorUserId,
+      profile = profile,
+      baseUUID = baseUUID,
+      metadata = metadata,
+      files = fms,
+      springfield = audioVideo.springfield
+    )
```

Comment:
I don't see much added value in using named values in this case. Having the same on the left and right for most arguments only confuses scala novices. Perhaps rename `fms` instead (`fileMedadataSeq`?).

Reply:
> I don't see much added value in using named values in this case. Having the same on the left and right for most arguments only confuses scala novices.

I mostly agree, but I was forced to. Since I introduced `bagId: BagId = UUID.randomUUID()` to `Deposit`, I wanted to take leverage of this default argument. Therefore I require the names of all parameters

https://github.com/DANS-KNAW/easy-split-multi-deposit/blob/eb1f9e279055ffafac7959089f0b0a19fa2f917d/src/main/scala/nl.knaw.dans.easy.multideposit/model/Deposit.scala#L43-L51

(btw. I moved the `bagId` from the `AddDepositProperties` class to here, since I needed the `bagId` to be used multiple times during the course of the application)

Reply:
> Perhaps rename fms instead (fileMedadataSeq?).

That name can be better indeed. I changed it to `fileMetadatas`. (first I wanted to use `fileMetadata`, since that is already plural, but then that would be confusing, since it is also single; therefore I just added an extra `s`)